app/pages/shows/_slug/index.vue

```diff
@@ -44,8 +44,20 @@
         <h3 class="pb-1 mb-4 text-center border-b border-current">
           Archived Shows
         </h3>
+        <div class="pt-4 pb-6 text-center change-order xsm:text-right">
+          <a id="bydate" ref="bydate" href="#" class="mr-2 selected change-order-button" @click.prevent="sortAirtime">
+            <i v-if="airtimeAsc" class="fa fa-sort-numeric-desc" aria-hidden="true" />
+            <i v-else class="fa fa-sort-numeric-asc" aria-hidden="true" />
+            Order by Air time
+          </a>
+          <a id="alphabetical" ref="alphabetical" class="change-order-button" href="#" @click.prevent="sortAlphabeticaly">
+            <i v-if="alphabeticAsc" class="fa fa-sort-alpha-asc" aria-hidden="true" />
+            <i v-else class="fa fa-sort-alpha-desc" aria-hidden="true" />
+            Order by Title
+          </a>
+        </div>
```

Comment:
If this block (line 47-58) is the same as for the show subpage then you may have a mini component for that so that we don't duplicate code.

Reply:
I think this is just a temporary quick-fix, don't know how the new UX/UI concept will handle this...